src/workflows/pyfr/common/steps/Simulation/Start/index.js

```diff
@@ -112,6 +112,7 @@ export default React.createClass({
       .then((resp) =>
         client.updateSimulationStep(this.props.simulation._id, 'Simulation', {
           view: 'run',
+          status: 'running',
```

Comment:
So the supported step statuses are enforce here:

https://github.com/Kitware/HPCCloud/blob/master/server/hpccloud/server/models/definitions.json#L61

If you want the add the 'running' state then it will need to be added to this list.